### PR TITLE
Added RockX

### PR DESCRIPTION
**Pull request by calvinzhou-rockx**

### CODE REVIEW

**Comment by calvinzhou-rockx** [comment]
https://polkadot.subscan.io/extrinsic/16091055-2

File: registry/polkadot.json [hunk 1]
```diff
@@ -30,5 +30,14 @@
...
         "isOrganization": false
+    },
+    {
+        "address": "14Y4s6V1PWrwBLvxW47gcYgZCGTYekmmzvFsK1kiqNH2d84t",
+        "name": "RockX | rockx.com",
+        "image": "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/rockx-logo.png",
+        "shortDescription": "RockX is a global blockchain node network and gateway for users to seamlessly access Web3, offering institutional-grade solutions and a full suite of developer tools for users to stake and build on all major Proof-of-Stake blockchains.",
+        "longDescription": "RockX is a global blockchain node network and gateway for users to seamlessly access Web3, offering institutional-grade solutions and a full suite of developer tools for users to stake and build on all major Proof-of-Stake blockchains.
+With its foundations rooted in staking, protocol research and infrastructure design, RockX has built countless access nodes for users to interact with over 20 layer 1 and 2 protocols — driving greater accessibility and furthering adoption of blockchain technologies.",
```
Comment (stepanLav) [inline registry/polkadot.json:40]: for making new line in text you can use "\n\n", like in that string:
```suggestion
        "longDescription": "RockX is a global blockchain node network and gateway for users to seamlessly access Web3, offering institutional-grade solutions and a full suite of developer tools for users to stake and build on all major Proof-of-Stake blockchains.\n\nWith its foundations rooted in staking, protocol research and infrastructure design, RockX has built countless access nodes for users to interact with over 20 layer 1 and 2 protocols — driving greater accessibility and furthering adoption of blockchain technologies.",
```

**Comment by stepanLav** [comment]
Hello @calvinzhou-rockx, thank you for request!
Could you fix longDescription text? Then, we will be able to merge that request.
Here you can find an example how it will looks like in the app:
<details>
  <summary>Spoiler!</summary>
Screenshot 2023-06-25 at 15 58 53
Screenshot 2023-06-25 at 15 58 57
Screenshot 2023-06-25 at 15 59 00

</details>

File: registry/polkadot.json (hunk 1 above)
Comment (stepanLav) [inline registry/polkadot.json:40]:
```suggestion
        "longDescription": "RockX is a global blockchain node network and gateway for users to seamlessly access Web3, offering institutional-grade solutions and a full suite of developer tools for users to stake and build on all major Proof-of-Stake blockchains.\n\nWith its foundations rooted in staking, protocol research and infrastructure design, RockX has built countless access nodes for users to interact with over 20 layer 1 and 2 protocols — driving greater accessibility and furthering adoption of blockchain technologies.",
```

**Comment by stepanLav** [comment]
Hello @calvinzhou-rockx, please make a fix in your request otherwise we will have to close your request, as I can't push changes directly in your branch

Screenshot 2023-07-06 at 22 35 44

**Comment by calvinzhou-rockx** [comment]
Hi Stepan,

I'm so sorry for missing your previous email, I've merged the changes from
your comments, please check again and let me know if it's OK

Thanks & Regards
Calvin

**Review by stepanLav — APPROVED** [review]

**Comment by stepanLav** [comment]
merged in another PR - https://github.com/novasamatech/opengov-delegate-registry/pull/64

**Pull request closed without merging**